CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased] - TBD
 ### Added
 - GitHub Action config file `.github/workflows/stale.yml`, which replaces StaleBot
- 
+
 ### Removed
-- GitHub config files `.github/stale.yml` and `.github/no-response.yml` 
+- GitHub config files `.github/stale.yml` and `.github/no-response.yml`
+
+### Changed
+- Converted Github issue templates into issue forms using TAML definition files
```

Comment:
Typo `TAML` -> `YAML`

Reply:
Thanks for catching this. It's now fixed in https://github.com/geoschem/HEMCO/pull/276/commits/7ef08fbba97cd9c9b99b6fdccdda6553efb4d7d7.

Reply:
Great!  Now good to merge.